examples/advanced/propagator/src/FairEveRecoTracksExampleEditor.h

```diff
@@ -0,0 +1,45 @@
+/********************************************************************************
+ *    Copyright (C) 2020 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairEveRecoTracksExampleEditor.h
+ *
+ *  Created on: 27 sie 2020
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef FAIREVERECOTRACKSEXAMPLEEDITOR_H_
+#define FAIREVERECOTRACKSEXAMPLEEDITOR_H_
+
+#include "FairEveCut.h"
+#include "FairEveRecoTracksExample.h"
+
+#include <TGedFrame.h>
+#include <memory.h>
```

Comment:
```suggestion
#include <memory>
```

STL headers have no `.h`, see https://en.cppreference.com/w/cpp/header/memory